test/test_browser.py

```diff
@@ -5417,6 +5417,17 @@ def test_audio_worklet_post_function(self, args):
   def test_audio_worklet_modularize(self, args):
     self.btest_exit('webaudio/audioworklet.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMODULARIZE=1', '-sEXPORT_NAME=MyModule', '--shell-file', test_file('shell_that_launches_modularize.html')] + args)
 
+  # We can't test playback, so don't need the data files, we're testing it builds and runs
```

Comment:
The comment says we don't need data files, but the code looks like to does copy them?

Reply:
Once I changed the test to btest_exit they were needed (because the test actually ran). I’ll make the change.